### ISSUE TITLE
[CI Problem] Flaky `test_ethosu/test_codegen.py`

### ISSUE THREAD

**Issue by masahi** [apache/tvm#9897]
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9815/1/
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9882/3/

@manupa-arm @leandron @lhutton1 @Mousius

**Comment by u99127** [comment]
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9882/3/


               # process is done if there is no data and the result is valid
                if not data:  # EOF
>                   break
E                   OSError: [Errno 28] No space left on device


We need to fix the CI node - it's run out of space.

**Comment by u99127** [comment]
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9815/1/pipeline

This looks like a catastrophic failure for all the tests for a first look , I'll wait for others who know more about this to comment.

**Comment by lhutton1** [comment]
> https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9815/1/pipeline
> 
> This looks like a catastrophic failure for all the tests for a first look , I'll wait for others who know more about this to comment.

This is #9815 which I believe is expected to be failing at the moment due to needing a CI update. cc @ekalda

**Comment by u99127** [comment]
Thanks @lhutton1  - so not a flaky test issue I suppose .

**Comment by ekalda** [comment]
> > https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-9815/1/pipeline
> > This looks like a catastrophic failure for all the tests for a first look , I'll wait for others who know more about this to comment.
> 
> This is #9815 which I believe is expected to be failing at the moment due to needing a CI update. cc @ekalda

Yes @lhutton1 is right, this patch is waiting for upstream CI image to be updated so that it would have the U-65 driver :)
